docs/customizing-templates.rst

```diff
@@ -33,6 +33,35 @@ title for any of the fields by using the ``scaffold.fields`` configuration
         // The rest of the fields to display here
     ]);
 
+Adding Controller Actions to utilize Crud Actions
+-------------------------------------------------
+
+It's easy to add an action to a controller that makes use of another
+*CrudView* action.
+
+This does use the template provided by the edit action:
+
+.. code-block:: php
+
+public function account() {
+    $this->Crud->mapAction('account', [
+        'className' => 'Crud.Edit',
+        'view' => 'edit',
+    ]);
+    return $this->Crud->execute(null, $this->Auth->user('id'));
+  }
+
+By default, it can be overwritten by providing a custom ``register.ctp``.
```

Comment:
Shouldn't it be `account.ctp`?

Reply:
nope. I do want it to render the crudview edit template in the account action with the user id of the currently logged in user. this is a real world example.
And this is the one not working without the patch.

Reply:
Ah, this statement is for the code block below it, not above.

Reply:
should make that more clear, use : instead of .